engine/ConsoleRunnerDotNet6/ConsoleRunnerDotNet6.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
All new files should have AGPL license notice on top. You can take the text from other csproj files.